index.bs

```diff
@@ -12346,12 +12365,62 @@ It is the responsibility of specifications using Web IDL to state
 which global environment (or, by proxy, which global object) each platform
 object is associated with.
 
+<div algorithm>
+  To <dfn export lt=new>create a new object implementing the interface</dfn> |interface|, with a
+  Realm |realm|, perform the following steps:
+
+    1.  Return the result of [=internally create a new object implementing the
+        interface|internally creating a new object implementing=] |interface|, with |realm| and
+        <emu-val>undefined</emu-val>.
+</div>
+
+<div algorithm>
+  To <dfn>internally create a new object implementing the interface</dfn> |interface|, with a
+  Realm |realm| and a JavaScript value |newTarget|, perform the following steps:
+
+    1.  Assert: |interface| is [=exposed=] in |realm|.
```

Comment:
Just as a note: this assert does NOT hold with the current definition of constructor behavior.  Last I checked, you could have a constructor from a Realm where the interface is exposed and a NewTarget from a Realm where the interface is _not_ exposed (at least with [SecureContext], where things can be same-origin yet different secure context state, last I checked) and the NewTarget Realm is what's getting passed in.  Yet another reason to not do that.  ;)